tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -287,6 +289,23 @@ void ordering_test_cases() {
         std::forward_list<SynthOrdered> b = {10, 20, 40};
         ordered_containers_test(a, a, b);
     }
+    { // slice

```

Comment:
This file hasn't rigorously sorted its `{ ... }` test blocks, so the order isn't critically important. However, these `slice` tests are being inserted in the middle of the container tests (between `forward_list` and `map`), which makes it harder to see the pattern of the container tests.

A reasonable place to move this test block would be between the `// Diagnostics Library` block (clause 19) and the `// filesystem::space_info` block (clause 29) below, as `slice` is Clause 26 Numerics.